x/superfluid/keeper/stake.go

```diff
@@ -32,24 +31,24 @@ func (k Keeper) GetSuperfluidOSMOTokens(ctx sdk.Context, denom string, amount sd
 	return k.GetRiskAdjustedOsmoValue(ctx, asset, decAmt.RoundInt())
 }
 
+func (k Keeper) GetExpectedDelegationAmount(ctx sdk.Context, acc types.SuperfluidIntermediaryAccount) sdk.Int {
+	// Get total number of Osmo this account should have delegated after refresh
+	totalSuperfluidDelegation := k.lk.GetPeriodLocksAccumulation(ctx, lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.ByDuration,
+		Denom:         acc.Denom + stakingSuffix(acc.ValAddr),
+		Duration:      time.Hour * 24 * 14, // should be a variable?
+	})
```

Comment:
Can we move this into its own function called `GetTotalSyntheticAssetsLocked(ctx, denom string)`?

Reply:
Sure, will do